engine/lib/sessions.php

```diff
@@ -286,6 +286,7 @@ function login(\ElggUser $user, $persistent = false) {
 	
 	if ($first_login) {
 		elgg_trigger_event('login:first', 'user', $user);
+		$user->first_login = time();
```

Comment:
maybe use `first_login_ts` and document the property with `ElggUser`

Reply:
not going to change... keeping name in line with other login metadata